src/async-wrap.h

```diff
@@ -33,6 +33,7 @@ namespace node {
   V(UDPWRAP)                                                                  \
   V(UDPSENDWRAP)                                                              \
   V(WRITEWRAP)                                                                \
+  V(HTTP)                                                                     \
```

Comment:
mind changing this to `HTTPPARSER` (that's exactly what it's being used for right?) and also ordering it alphabetically?